native-link-store/tests/filesystem_store_test.rs

```diff
@@ -865,4 +869,66 @@ mod filesystem_store_tests {
 
         Ok(())
     }
+
+    #[tokio::test]
+    async fn get_part_timeout_test() -> Result<(), Error> {
+        let large_value = std::iter::repeat("x").take(1024).collect::<String>();
+        let digest1 = DigestInfo::try_new(HASH1, large_value.len())?;
```

Comment:
nit (should not hold up PR, because it is a test): I know this is only a test, I'm curious why this variable name was chosen. It's your code so I shouldn't dictate, but my intuition would've been to name this variable `many_x_digest` or something similar to represent the long string of x's for a few reasons. 

1. `digest` is a moderately common function name in Rust.
2. `digest1` is slightly non-descript. Not totally, it is a digest, and the first one. 

On the other hand, it's not that important because it doesn't really matter what the value is, as the important point is that it is a large string.